api/user_api.js

```diff
@@ -0,0 +1,78 @@
+let db = require('../db/database_mongo');
+
+/**
+ * @api {get} /users Retrieve All Users
+ * @apiName GetUsers
+ * @apiGroup User
+ *
+ * @apiSuccessExample Success-Response:
+ *     HTTP/1.1 200 OK
+ *     [
+ *       {user-object},
+ *       {user-object}
+ *     ]
+ */
+exports.getUsers = function (request) {
+  return new Promise(resolve => {
+    db.getUsers(request).then(results => {
+      resolve(results);
+    });
+  });
+};
+
+/**
+ * @api {get} /users/:id Retrieve User
+ * @apiName GetUser
+ * @apiGroup User
+ *
+ * @apiParam {Number} id Users unique ID.
+ *
+ * @apiSuccess {String} _id Id of the User.
+ * @apiSuccess {String} familyName Last Name of the User.
+ * @apiSuccess {String} imageUrl URL to the Users Gravatar.
+ * @apiSuccess {String} name Name of the User.
+ * @apiSuccess {String} googleId GoogleId of the User.
+ * @apiSuccess {String} email Email of the User.
+ * @apiSuccess {String} givenName First name of the User.
+ *
+ * @apiSuccessExample Success-Response:
```

Comment:
yeah, same here maybe we can create a test account to use, but don't want this on our repo